app/serializers/group_serializer.rb

```diff
@@ -57,4 +57,8 @@ def has_discussions
   def has_multiple_admins
     object.admins.count > 1
   end
+
+  def cover_photo
```

Comment:
So... how crazy do you think it would be to put something like

```
default_scope { includes(:default_cover_photo) }
```

into group.rb

I've always ended up regretting using default_scope.